templates/eks-protection-stack.yml

```diff
@@ -0,0 +1,536 @@
+            comment: Lambda does not need reserved concurrent executions.
+          - id: CKV_AWS_116
+            comment: DLQ not needed, as Lambda function only triggered by CloudFormation events.
+          - id: CKV_AWS_117
+            comment: Lambda does not need to communicate with VPC resources.
+          - id: CKV_AWS_173
+            comment: Environment variables are not sensitive.
+    Properties:
+      Environment:
+        Variables:
+          project_name: !Ref CodeBuildProjectName
+          artifact_bucket: !Ref StagingS3Bucket
+          lambda_switch_role: !Ref EKSExecutionRoleName
+      Handler: lambda_function.lambda_handler
+      MemorySize: 128
+      Role: !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:role/${EKSExecutionRoleName}
+      Runtime: python3.11
+      Timeout: 300
+      FunctionName: crowdstrike-abi-eks-events-function
+      Code:
+        S3Bucket: !Ref StagingS3Bucket
+        S3Key: !Sub ${SourceS3BucketNamePrefix}/lambda_functions/packages/eks-new-clusters/lambda.zip
+  EKSInvokeLambdaPermission: 
+    Type: AWS::Lambda::Permission
+    Properties: 
+      FunctionName: !Ref EKSLambda
+      Action: "lambda:InvokeFunction"
+      Principal: "events.amazonaws.com"
+      SourceArn: !GetAtt EKSRule.Arn
+  ExistingEKSLambda:
+    Type: AWS::Lambda::Function
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W89
+            reason: Lambda custom resource only run during stack lifecycle events.
+          - id: W92
+            reason: Lambda custom resource only run during stack lifecycle events.
+      checkov:
+        skip:
+          - id: CKV_AWS_115
+            comment: Lambda does not need reserved concurrent executions.
+          - id: CKV_AWS_116
+            comment: DLQ not needed, as Lambda function only triggered by CloudFormation events.
+          - id: CKV_AWS_117
+            comment: Lambda does not need to communicate with VPC resources.
+          - id: CKV_AWS_173
+            comment: Environment variables are not sensitive.
+    Properties:
+      Environment:
+        Variables:
+          project_name: !Ref CodeBuildProjectName
+          artifact_bucket: !Ref StagingS3Bucket
+          lambda_switch_role: !Ref EKSExecutionRoleName
+      Handler: lambda_function.lambda_handler
+      MemorySize: 128
+      Role: !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:role/${EKSExecutionRoleName}
+      Runtime: python3.11
+      Timeout: 300
+      FunctionName: crowdstrike-abi-eks-init-function
+      Code:
+        S3Bucket: !Ref StagingS3Bucket
+        S3Key: !Sub ${SourceS3BucketNamePrefix}/lambda_functions/packages/eks-existing-clusters/lambda.zip
+  # Trigger Lambda Function
+  TriggerExistingEKSLambda:
+    Type: 'Custom::TriggerLambda'
+    DependsOn:
+      - SMFalconAPIKey
+    Properties:
+      ServiceToken: !GetAtt
+        - ExistingEKSLambda
+        - Arn
+
+  # CodeBuild Project to deploy Falcon Operator and Sensor
+  EKSCodeBuild:
```

Comment:
which linter does this flag apply?  I do not see it in cfnlint or checkov

Reply:
but the default cmk will suffice

Reply:
This is coming from cfn_nag. Could you please add an exception to avoid flagging this in future. No value for `EncryptionKey` defaults to `the managed CMK for Amazon Simple Storage Service (Amazon S3)`